src/utils.ts

```diff
@@ -113,3 +114,20 @@ export const getAttachments = async (attachments: attachments): Promise<Attachme
 export const isErrorLog = (message: string): boolean => {
   return message.toLowerCase().includes('error');
 };
+
+// https://playwright.dev/docs/1.17/api/class-testresult#test-result-status
+export const convertToRpStatus = (status: TestStatus): string => {
+  const isRpStatus = Object.values(STATUSES).includes(<STATUSES>status);
+
+  if (isRpStatus) {
+    return status;
+  }
+
+  switch (status) {
+    case 'timedOut': {
+      return STATUSES.FAILED;
+    }
+    default:
+      return STATUSES.FAILED;
```

Comment:
Does we really needed switch case construction here? Look like you can omit this util and check check status directly in the `reporter` 
`STATUSES.status && status || STATUSES.FAILED`